frontend/src/views/instructors/editable-instructors-list.js

```diff
@@ -42,6 +56,14 @@ function EditableInstructorsList(props) {
         );
     }
 
+    const workingInstructors = new Set(
```

Comment:
What is this variable name trying to say? Pick a better one.

Reply:
A simple hash will do just as well as a set and be simpler to understand.

Reply:
> A simple hash will do just as well as a set and be simpler to understand.

What do you mean?

Reply:
`instructorCurrentlyAssignedHash[utorid] = true`